internal/manifests/gateway_tenant_sidecars_test.go

```diff
@@ -0,0 +1,438 @@
+package manifests
+
+import (
+	"testing"
+
+	monitoringv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
+	"github.com/stretchr/testify/require"
+
+	lokiv1beta1 "github.com/ViaQ/loki-operator/api/v1beta1"
+	"github.com/ViaQ/loki-operator/internal/manifests/internal/gateway"
+
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+func Test_ApplyGatewayDefaultsOptions(t *testing.T) {
```

Comment:
```suggestion
func TestApplyGatewayDefaultsOptions(t *testing.T) {
```
For consistency in naming everywhere.